.gitattributes

```diff
@@ -4,3 +4,5 @@ phpunit.xml export-ignore
 *.md export-ignore
 /tests export-ignore
 /packages-tests export-ignore
+
+.phpunit.result.cache
```

Comment:
This should have been in `.gitignore` instead, right?

Reply:
I see you already removed the file and added it to `.gitignore`, but this entry is still left. I fixed it: #18